cdk/src/open-data-platform/data-plane/data-import/write-demographic-data.handler.ts

```diff
@@ -1,149 +1,211 @@
+const { ignore } = require('stream-json/filters/Ignore');
+const { pick } = require('stream-json/filters/Pick');
+const { parser } = require('stream-json/Parser');
+const { streamArray } = require('stream-json/streamers/StreamArray');
+const Batch = require('stream-json/utils/Batch');
+
 const S3 = new AWS.S3();
 
-const GEO_ID = 'GEOID';
-const RACE_TOTAL = 'RaceTotal';
-const WHITE_POPULATION = 'Estimate!!Total:!!White alone';
-const BLACK_POPULATION = 'Estimate!!Total:!!Black or African American alone';
+// Number of rows to write at once.
+const BATCH_SIZE = 10;
 
 /**
- * Inserts all rows into the demographics table.
+ *  Writes rows into the demographics table.
+ * @param rdsService: RDS service to connect to the db.
+ * @param rows: Rows to write to the db.
  */
-async function insertRows(db: Queryable, rows: DemographicsTableRow[]): Promise<any[]> {
-  // TODO(breuch): Replace with geom from new file.
-  // This is a random polygon taken from an unmapped s3 file.
-  const geometry = sql.__dangerous__rawValue(
-    "ST_GeometryFromText('POLYGON((-122.76199734299996 " +
-      '47.34350314200003, -122.76248119999997 47.342854930000044, ' +
-      '-122.76257080699997 47.34285604200005, -122.76259517499994 ' +
-      '47.34266843300003, -122.76260856299996 47.34256536000004, ' +
-      '-122.76286629299995 47.34250175600005, -122.76295867599998 ' +
-      '47.34242223000007, -122.76328431199994 47.34202772300006, ' +
-      '-122.763359015 47.34188063000005, -122.76359184199998 ' +
-      '47.34185498100004, -122.76392923599997 47.34181781500007, ' +
-      '-122.76392897999995 47.34183731100006, -122.76390878299998 ' +
-      '47.341839519000075, -122.76390520999996 47.34211214900006, ' +
-      '-122.76390440899996 47.342173262000074, -122.763924584 ' +
-      '47.34217272700005, -122.763921139 47.342407894000075, ' +
-      '-122.76391819399998 47.342471904000035, -122.76390565499997 ' +
-      '47.34257032900007, -122.76390536899999 47.342571914000075, ' +
-      '-122.76388240099999 47.34267173100005, -122.76388191899997 ' +
-      '47.34267343500005, -122.76386218299996 47.34273572600006, ' +
-      '-122.76383851199995 47.34279737900005, -122.76383805499995 ' +
-      '47.34279846700008, -122.76380443699998 47.34287143500006, ' +
-      '-122.76376811299997 47.34293827000005, -122.763727072 ' +
-      '47.343003846000045, -122.76372118599994 47.34301258000005, ' +
-      '-122.763375968 47.34352033700003, -122.76328781999996 ' +
-      '47.343519239000045, -122.76310988499995 47.343517020000036, ' +
-      "-122.76199734299996 47.34350314200003))')",
-  );
+async function insertBatch(
+  rdsService: RDSDataService,
+  rows: SqlParametersList[],
+): Promise<RDSDataService.BatchExecuteStatementResponse> {
+  const batchExecuteParams: BatchExecuteStatementRequest = {
+    database: process.env.DATABASE_NAME ?? 'postgres',
+    parameterSets: rows,
+    resourceArn: process.env.RESOURCE_ARN ?? '',
+    schema: 'public',
```

Comment:
nit: make const. I use SCHEMA in the other handler

Reply:
Done.